### ISSUE TITLE
Redirect Loop when using client-cookie for session data that exceeds 4096bytes

### ISSUE THREAD

**Issue by jahed** [pingidentity/mod_auth_openidc#92]
Just had this issue on our production systems for the passed week or so.

When using Google's OpenID Connect and `OIDCSessionType client-cookie`, the `mod_auth_openidc_session` cookie can now exceed 4096bytes which is the cookie size limit on Google Chrome and most other web browsers. This means that the cookie doesn't get set in the browser and the server will think the redirected request needs to be authorised again, causing a redirect loop. Not every user exceeds the limit but some do.

Not sure what the cause is myself. Previously cookies were only around 30bytes. Maybe Google changed something on their end?

For now, using `server-cache` as the `OIDCSessionType` is a good enough workaround for us.

**Comment by zandbelt** [comment]
IMHO cookies previously couldn't have been around 30 bytes. If you don't need all the attributes from the profile, you could reduce the requested scopes. Otherwise `server-cache` is your only option.

**Comment by jahed** [comment]
Now that you mention it, the 30 byte variant I reported was just a session key cookie when we switched over to `server-cache`.

We're using `OIDCScope "openid email profile"` so it's possible Google may have just increased the amount of information they provide for the `profile` in recent weeks. If that is the case, for users to avoid such problems, it might just be worth documenting the limitations of `client-cookie` in [/auth_openidc.conf](https://github.com/pingidentity/mod_auth_openidc/blob/master/auth_openidc.conf). I've made a pull request for this (PR #93)

Thanks,
